README.md

```diff
@@ -705,6 +705,46 @@ Chainweb.Pact.listen(request_key, network_id: :testnet04, chain_id: 1)
    tx_id: 4_272_500
  }}
 ```
+### SPV endpoint
+
+Retrieves a SPV proof of a cross chain transaction. Request must be sent to the chain where the transaction is initiated.
+
+```elixir
+Kadena.Chainweb.Pact.spv(payload_opts, network_opts \\ [network_id: :testnet04, chain_id: 0])
+```
+
+**Parameters**
+
+- `payload_opts`: Payload options. Keyword list with:
+
+  - `request_key` (required): String value. Request Key of an initiated cross chain transaction at the source chain.
+
+  - `target_chain_id` (required): String-encoded integer from 0 to 19. Target chain id of the cross chain transaction.
+
+
+- `network_opts`: Network options. Keyword list with:
+
+  - `network_id` (required): Allowed values: `:testnet04` `:mainnet01`.
+  - `chain_id` (required): Allowed values: integer or string-encoded integer from 0 to 19.
+
+  Defaults to `[network_id: :testnet04, chain_id: 0]` if not specified.
+
+**Example**
+
+```elixir
+alias Kadena.Chainweb
+
+payload_opts = [request_key: "VB4ZKobzuo5Cwv5LT9kWKg-34u7KZ0Oo84jnIiujTGc", target_chain_id: "1"]
+
+Chainweb.Pact.spv(payload_opts, network_id: :testnet04, chain_id: 1)
+
+{:ok,
+ %Kadena.Chainweb.Pact.SPVResponse{
+   response:
```

Comment:
@EdwinGuayacan Small tweak: Can we switch from `response` to `proof` in the SPVResponse module?